db/src/tx.rs

```diff
@@ -51,6 +51,7 @@ use std::collections::{
     BTreeSet,
     VecDeque,
 };
+use std::rc::Rc;
```

Comment:
I haven't been doing this elsewhere, including in this file, so I'm going to keep it this way.  Happy to change everywhere if we're committed.

I have been putting lines between *sections*, i.e., between `std::`, crates in this workspace, and modules in this crate.